### DISCUSSION TITLE
Popover not occupying the  full width of trigger

### DISCUSSION THREAD

**Discussion by RajaTheKing826** [adobe/react-spectrum#7048] [Q&A] [answered]
I've developed a combobox using react aria components, but im facing an issue with it which is my popover is not occupying the full width of trigger. How can i do this? 

Here is the codesandbox link -> https://codesandbox.io/p/sandbox/j4wsct?file=%2Fsrc%2FRACComboBox.tsx%3A42%2C1

**Comment by reidbarber** [comment, marked as answer]
Use the `--trigger-width` css variable: https://codesandbox.io/p/sandbox/react-aria-components-combobox-forked-6744s5?file=%2Fsrc%2Fstyles.css%3A8%2C1-9%2C1&workspaceId=7453408d-4fc8-4591-8707-0c83aeb1ce66

**Comment by federicocappellotto97** [comment]
I can't find that css variable :( what am i missing?
"react-aria-components": "1.6.0"

```
<DialogTrigger>
   <button>trigger</button>
      <Popover>
         <Dialog>
                        ...
         </Dialog>
      </Popover>
</DialogTrigger>
```

**Reply by reidbarber** [reply]
It only exists in ComboBox, DatePicker, DateRangePicker, Menu, and Select currently. Seems like something we could add to DialogTrigger though.
